tuning/data/data_preprocessing_utils.py

```diff
@@ -43,12 +44,28 @@ def get_data_collator(
             Whether train Dataset is tokenized or not
         max_seq_length: int
             Max sequence length expected
+        instruction_template: str
+            str representing the human response in a chat template
 
     Returns:
         Callable
             Callable collator to be leveraged by the trainer.
     """
 
+    if response_template and instruction_template:
+        # response_template_ids = tokenizer.encode(
+        #     response_template, add_special_tokens=False
+        # )[2:]
+        # intruction_template_ids = tokenizer.encode(
+        #     instruction_template, add_special_tokens=False
+        # )[2:]
```

Comment:
Thanks for passing the strings directly rather tokens, shall we remove these comments then?